packages/manager/apps/pci-rancher/tailwind.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    '../../../manager-components/src/**/*.{js,jsx,ts,tsx}',
```

Comment:
**nitpick:** We cannot include `manager-components` here in every consumer of `manager-components`. That should be an independent package.

Reply:
The comment will be handled with jira MANAGER-14313.